src/Umbraco.Web.Common/Umbraco.Web.Common.csproj

```diff
@@ -21,6 +21,9 @@
     </ItemGroup>
 
     <ItemGroup>
+      <PackageReference Include="Microsoft.AspNet.Identity.Core" Version="2.2.3" />
```

Comment:
**issue:** This is part of the framework now (which is why there is no v3+ package). If you're using .NET 5, you'll need a `FrameworkReference` in here instead.

Reply:
Oh, I didn't mean for this change to happen. Will revert!